README.md

```diff
@@ -27,10 +25,23 @@ module "kms" {
   location           = "europe"
   keyring            = "sample-keyring"
   keys               = ["foo", "spam"]
-  set_owners_for     = ["foo", "spam"]
-  owners = [
-    "group:one@example.com,group:two@example.com",
-    "group:one@example.com",
+
+  key_opts = [
```

Comment:
IIUC, this for overriding `key_opts_default` for individual keys, maybe we can simplify this to a map like keyName => {options}. WDYT?
```tf
key_opts = [ 
  foo = {rotation_period = "100000s"},
  bar = {algorithm = "EXTERNAL_SYMMETRIC_ENCRYPTION"}
]
```

Reply:
Doesn't simplify much, and I prefer more the behavior seen in google kubernetes module...

Reply:
This field seems to be redundant with `keys`. I suggest we either:

1. Make `keys` a list of objects (including name and optional parameters).
2. Make this into a map of `key_name -> object()`.